tests/unittests/Foundation/NSURLTests.m

```diff
@@ -62,7 +62,7 @@ void testNSURLMethod(SEL selector, NSURL* input, id argument, NSURL* expected) {
                     nil,
                     [[NSURL alloc] initWithString:@"http://www.test.com/home.page/asdf"]);
 
-    testNSURLMethod(@selector(URLByDeletingPathExtension), testURL7, nil, [[NSURL alloc] initWithString:@"http://www.test.com/"]);
+    testNSURLMethod(@selector(URLByDeletingPathExtension), testURL7, nil, [[NSURL alloc] initWithString:@"http://www.test.com/.foo"]);
```

Comment:
>testNSURLMethod(@selector(URLByDeletingPathExtension), testURL7, nil, [[NSURL alloc] initWithString:@"http://www.test.com/.foo"]); [](start = 4, length = 130)

is the old one still a valid test case? Should we switch to paramterized tests here?